README.rst

```diff
@@ -1,25 +1,47 @@
+.. raw:: html
+
+   <p align="center">
+     <img width="450" height="200" src=docs/tialab_logo.png>
+   </p>
+
 ===========
 TIA Toolbox
 ===========
 
+Computational Pathology Toolbox developed by TIA Lab
+
+Please try
 
+::
 
+    python -m tiatoolbox -h
 
+Getting Started
+===============
 
+First, install OpenSlide `here <https://openslide.org/download/>`__. Then, create and
+activate the conda environment:
 
-Computational pathology toolbox developed by TIA Lab.
+::
 
+    conda env create --name tiatoolbox --file requirements.conda.yml
+    conda activate tiatoolbox
 
+tiatoolbox --help
+=======================
 
-Features
---------
+::
 
-* TODO
+    usage: tiatoolbox [-h] [--version] [--verbose VERBOSE]
+                     {slide_info}
+                     ...
 
-Credits
--------
+    positional arguments:
+      {slide_info}
+        slide_info          usage: python -m tiatoolbox slide_info -h
 
-This package was created with Cookiecutter_ and the `audreyr/cookiecutter-pypackage`_ project template.
+    optional arguments:
+      -h, --help            show this help message and exit
+      --version             show program's version number and exit
+      --verbose VERBOSE
 
```

Comment:
May want to provide links to contributing, license (when we sort it) etc.

Reply:
Sure, once we have sorted this out we should add this to docstring.